### PR TITLE
Add Postgres Container

### PR DESCRIPTION
**Pull request by Sironheart**

### CODE REVIEW

File: src/Container/PostgresContainer.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class PostgresContainer extends Container
+{
+    private function __construct(string $version, string $rootPassword)
+    {
+        parent::__construct('postgres:' . $version);
+        $this->withEnvironment('POSTGRES_PASSWORD', $rootPassword);
+        $this->withWait(new WaitForExec(["pg_isready", "-h", "127.0.0.1"]));
+    }
+
+    public static function make(string $version = 'latest', string $mysqlRootPassword = 'root'): self
```
Comment (shyim) [inline src/Container/PostgresContainer.php:18]: mysqlRootPassword?

File: src/Container/PostgresContainer.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        return new self($version, $mysqlRootPassword);
+    }
+
+    public function withPostgresUser(string $username): self
+    {
+        $this->withEnvironment('POSTGRES_USER', $username);
```
Comment (shyim) [inline src/Container/PostgresContainer.php:25]: the user don't have an password?

Comment (Sironheart) [reply src/Container/PostgresContainer.php:25]: The User, that is given, will be the only accessible user. There is no thing as a root user for postgres docker container

File: src/Container/PostgresContainer.php (hunk 1 above)
Comment (Sironheart) [reply src/Container/PostgresContainer.php]: fixed it

**Pull request merged**